projects/1.16/assets/tinkers-construct/tconstruct/book/tinkers_gadgetry/zh_cn/exploration/earthslime.json

```
@@ -0,0 +1,13 @@
+{
+  "title": "大地史莱姆岛",
+  "image": {
+	"file": "tconstruct:book/images/islands/earthslime.png",
+    "width": 180,
+    "height": 84
+  },
+  "text": [
+    {
+      "text": "大地史莱姆岛有时在主世界的海洋被发现，看起来他们并不像是碧空黏液的亲戚们那么不常见， 这也并不是寻找这寻常的绿色黏液的唯一途径，也许是这些草使得他们更常见。"
```

```suggestion
      "text": "大地史莱姆岛有时可以在主世界的海洋找到，和它们的碧空黏液变种相比，这些岛屿看起来没那么奇异。 这也并不是寻找这寻常的绿色黏液的唯一途径，不过也许是这些草使得它们更常见。"
```
这段话太怪了。alien我倾向于翻译成“异国的，奇异的”。